root/static/scripts/edit/components/edit/Diff.js

```diff
@@ -21,7 +21,7 @@ export type DiffProps = {
 
 type Props = {
   ...DiffProps,
-  +split: string,
+  +split?: string,
```

Comment:
Hmm. Isn't there always a split, since a default is defined below? Maybe that doesn't work like that, I'm not sure. @mwiencek?

Reply:
default values will just assign default values to props. But flow checks to see if the files which is rendering the components to make sure all the props are passed. At least that's what I think it does 😓